crates/schema-impl/src/schemas_impl.rs

```diff
@@ -198,12 +198,13 @@ impl ServiceSchemas {
     ) -> HashMap<String, MethodSchemas> {
         svc_desc
             .methods()
+            .filter(|descriptor| method_meta.contains_key(&descriptor.name().to_string()))
```

Comment:
Till had a nice suggestion in https://github.com/restatedev/restate/pull/932 to replace this with:

```rs
svc_desc
    .methods()
    .flat_map(|descriptor| {
        let method_name = descriptor.name().to_string();
        method_meta.get(&method_name).map(|metadata| {
            (
                method_name,
                MethodSchemas::new(descriptor, metadata.input_fields_annotations.clone()),
            )
        })
    })
        .collect()
```

still passes the tests with that.

Reply:
Will apply to next PR